test/test.py

```diff
@@ -45,40 +54,68 @@ def test_initialization(self):
             self.assertTrue(isinstance(self.config.email_subject, basestring))
             self.assertTrue(isinstance(self.config.email_body, basestring))
 
+# This dummy config prevents changes to the user-defined config
+# from changing their results when doing testing.
+MOCK_DATA = {
+    'package_urls': [
+        "https://www.nuget.org/packages/SendGrid",
+        "https://www.nuget.org/packages/SendGrid.CSharp.HTTP.Client",
+        "https://www.npmjs.com/package/sendgrid",
+        "https://www.npmjs.com/package/sendgrid-rest",
+        "https://packagist.org/packages/sendgrid/sendgrid",
+        "https://packagist.org/packages/sendgrid/php-http-client",
+        "https://pypi.python.org/pypi/sendgrid",
+        "https://pypi.python.org/pypi/python_http_client",
+        "https://pypi.python.org/pypi/open_source_library_data_collector",
+        "https://rubygems.org/gems/sendgrid-ruby",
+        "https://rubygems.org/gems/ruby_http_client",
+    ],
+    'github_config': {
+        'user': 'sendgrid',
+        'repositories': [
+            "sendgrid-csharp",
+            # "smtpapi-csharp"
+        ]
+    }
+}
+
 
 class TestDBConnector(unittest.TestCase):
+
     def setUp(self):
         if os.environ.get('TRAVIS') == None:
             self.db = DBConnector()
 
-    def test_add_and_delete_data(self):
+    def test_add_and_delete_github_data(self):
         if os.environ.get('TRAVIS') == None:
             github_data_import = GitHubData(
-                                    date_updated=datetime.datetime.now(),
-                                    language='repo_name',
-                                    pull_requests=0,
-                                    open_issues=0,
-                                    number_of_commits=0,
-                                    number_of_branches=0,
-                                    number_of_releases=0,
-                                    number_of_contributors=0,
-                                    number_of_watchers=0,
-                                    number_of_stargazers=0,
-                                    number_of_forks=0
-                                    )
+                date_updated=datetime.datetime.now(),
+                language='repo_name',
+                pull_requests=0,
+                open_issues=0,
+                number_of_commits=0,
+                number_of_branches=0,
+                number_of_releases=0,
+                number_of_contributors=0,
+                number_of_watchers=0,
+                number_of_stargazers=0,
+                number_of_forks=0
+            )
             res = self.db.add_data(github_data_import)
             self.assertTrue(isinstance(res, GitHubData))
```

Comment:
`self.assertIsInstance` ?

* https://docs.python.org/3/library/unittest.html?#unittest.TestCase.assertIsInstance

Reply:
@42B thanks for catching these items - with that said, these lines fall outside the scope of the topics for this active PR, if you're interested, maybe those changes could go into a separate PR